quark/utils/graph.py

```diff
@@ -191,30 +192,37 @@ def call_graph(call_graph_analysis, output_format="png"):
         )
 
     if second_call != second_api:
-        dot.edge(
-            f"{parent_function.full_name}",
-            f"{second_wrapper[-1].full_name}",
-            "Second Call",
-            fontname="Courier New",
-        )
+        try:
+            dot.edge(
+                f"{parent_function.full_name}",
+                f"{second_wrapper[-1].full_name}",
+                "Second Call",
+                fontname="Courier New",
+            )
 
-        dot.edge(
-            f"{second_wrapper[0].full_name}",
-            f"{second_api.full_name}",
-            "calls",
-            fontname="Courier New",
-        )
+            dot.edge(
+                f"{second_wrapper[0].full_name}",
+                f"{second_api.full_name}",
+                "calls",
+                fontname="Courier New",
+            )
+        except IndexError:
+            pass
     else:
```

Comment:
How about checking if `second_wrapper` is empty rather than using try-except? It could simplify the code.

Reply:
Done.